corona-warn-app/cwa-app-android
# Certificate details missing (EXPOSUREAPP-12934)

This PR addresses EXPOSUREAPP-[12934](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12934).
See ticket for details/screenshots.

How to test:

- Create RC or VC
- Revoke certificate
- Scan certificate

Wait until certificate gets recognized as Revoked and a corresponding message should appear under the qr code.

Info: ask me for a testqr if you want
